.github/workflows/phpunit-tests.yml

```diff
@@ -124,16 +125,29 @@ jobs:
   # - Submit the test results to the WordPress.org host test results.
   # - todo: Configure Slack notifications for failing tests.
   test-php:
-    name: ${{ matrix.php }}${{ matrix.multisite && ' multisite' || '' }}${{ matrix.memcached && ' with memcached' || '' }} on ${{ matrix.os }}
+    name: ${{ matrix.php }}${{ matrix.multisite && ' multisite' || '' }}${{ matrix.slow56 && ' slow tests' || '' }} ${{ matrix.memcached && ' with memcached' || '' }} on ${{ matrix.os }}
     needs: setup-wordpress
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
         php: [ '5.6.20', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0' ]
         os: [ ubuntu-latest ]
         memcached: [ false ]
+        slow56: [ false ]
```

Comment:
Naming things: once 5.6 is dropped another version will be the slow one, a more generic name would be dandy: `split_slow`.

Reply:
While I don't think there will be any benefit to doing this on PHP 7.0 (the setup time for the job most likely exceeds the time saving) and I think we'll just remove this approach when support for 5.6 is dropped, that's a good idea regardless.